Files/Filesystem/FilesystemOperations/Helpers/FilesystemHelpers.cs

```diff
@@ -226,6 +229,7 @@ public async Task<ReturnResult> DeleteItemAsync(IStorageItemWithPath source, boo
             sw.Start();
 
             IStorageHistory history = await filesystemOperations.DeleteAsync(source, banner.Progress, banner.ErrorCode, permanently, cancellationToken);
```

Comment:
Should the banner.Progress argument be null here too?

Reply:
It's for one item only, so it will get `100.0` reported anyway